scripts/lint/ourlint.js

```diff
@@ -199,6 +205,37 @@ const TESTS = {
       return `Invalid URLs:\n\n${invalidUrls.join('\n\n')}`;
     }
   },
+
+  /* Check if all licenses are valid SPDX identifiers */
+  async checkLicense(data) {
+    const spdxLicenseIds = JSON.parse(
```

Comment:
Shouldn't this be a `Set`?

Reply:
Sorry, I didn't get you. What do you mean Set? It reads the list from the JSON file.

Reply:
To convert the list to a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) because SPDX license identifiers are unique values. Mainly because the intent is clear and its faster, especially for large sets:

> The [has](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/has) method checks if a value is in the set, using an approach that is, on average, quicker than testing most of the elements that have previously been added to the set. In particular, it is, on average, faster than the [Array.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method when an array has a length equal to a set's size.

Reply:
Sounds good. Done.